metricflow/dataset/convert_semantic_model.py

```diff
@@ -423,8 +424,8 @@ def create_sql_source_data_set(self, semantic_model: SemanticModel) -> SemanticM
         all_entity_instances: List[EntityInstance] = []
 
         all_select_columns: List[SqlSelectColumn] = []
-        from_source_alias = IdGeneratorRegistry.for_class(self.__class__).create_id(f"{semantic_model.name}_src")
-
+        # from_source_alias = IdGeneratorRegistry.for_class(self.__class__).create_id(f"{semantic_model.name}_src")
+        from_source_alias = PrefixIdGenerator.create_next_id(IdPrefix.SEMANTIC_MODEL_SOURCE).str_value
```

Comment:
This is a big change since the constant now formats in `src` instead of `{semantic_model.name}_src`